src/H5Zdeflate.c

```diff
@@ -23,7 +23,7 @@
 #define H5_ZLIB_HEADER "zlib.h"
 #endif
 #if defined(H5_ZLIB_HEADER)
-#include H5_ZLIB_HEADER /* "zlib.h" */
+#include H5_ZLIB_HEADER "zlib.h"
```

Comment:
I don't think you want both, and why does H5_ZLIB_HEADER fail since it should be exactly the same as the text you added?

Reply:
direct_chunk.c has the same include block - does it fail?

Reply:
The reason it is not hard-coded is that the header file could be overridden with something different.

Reply:
> direct_chunk.c has the same include block - does it fail?

No. (I guess build process failed before it reached direct_chunk.c).

Reply:
> I don't think you want both, and why does H5_ZLIB_HEADER fail since it should be exactly the same as the text you added?

I think the problem is escape character: `\"zlib.h\"`. How can we fix it?

Reply:
That was passed in as a define so that needs fixed